ballista/rust/core/Cargo.toml

```diff
@@ -42,6 +42,8 @@ tokio = "1.0"
 tonic = "0.6"
 uuid = { version = "0.8", features = ["v4"] }
 chrono = { version = "0.4", default-features = false }
+clap = "2"
+parse_arg = "0.1.3"
```

Comment:
would be nice if we can avoid depending on CLI related dependencies like clap and parse_arg in the core, but I think it's a minor issue we can clean up as follow ups later.